scene/animation/animation_blend_tree.h

```diff
@@ -36,6 +36,8 @@
 class AnimationNodeAnimation : public AnimationRootNode {
 	GDCLASS(AnimationNodeAnimation, AnimationRootNode);
 
+	StringName backward = PNAME("backward"); // Only used by pingpong animation.
```

Comment:
Isn't this for translation of names for the editor?

Reply:
Saw that it was used to declare a property not just as a placeholder so might have been wrong

Reply:
It is fine to remove PNAME, as the property does not appear in the editor, thanks.